mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/meta/MetaPersistManagerSample.java

```diff
@@ -64,15 +63,14 @@ public class MetaPersistManagerSample implements MetaPersistManager {
     public void init() throws MQClientException, RemotingException, InterruptedException, MQBrokerException {
         defaultMQAdminExt = MqFactory.buildDefaultMQAdminExt("MetaLoad", serviceConf.getProperties());
         defaultMQAdminExt.start();
-        refreshMeta();
         scheduler = new ScheduledThreadPoolExecutor(1, new ThreadFactoryImpl("refreshMeta"));
         scheduler.scheduleWithFixedDelay(() -> {
             try {
                 refreshMeta();
             } catch (Throwable t) {
                 logger.error("", t);
             }
-        }, 5, 5, TimeUnit.SECONDS);
+        }, 0, 5, TimeUnit.SECONDS);
```

Comment:
Delaying through time will result in slower startup times, which is not conducive to future serverless resilience capabilities. It may be possible to solve the startup order problem through bean dependencies.

Reply:
`NotifyManager` and `MetaPersistManagerSample` are dependent on each other.

Reply:
NotifyManager should be ready before MetaPersistManagerSample#refreshMeta. This change causes refreshMeta to come earlier, increasing the chance of an error